integration-test/pom.xml

```diff
@@ -124,84 +83,152 @@
+                            <finalName>unwanted</finalName>
+                            <classifier>unwanted</classifier>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
-                    <compilerVersion>1.8</compilerVersion>
-                    <showWarnings>true</showWarnings>
-                    <encoding>UTF-8</encoding>
-                </configuration>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.33.0</version>
+                <executions>
+                    <execution>
+                        <id>prepare-it-container</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>start</goal>
+                        </goals>
+                        <configuration>
+                            <showLogs>true</showLogs>
+                            <images>
+                                <image>
+                                    <name>${docker.container.repo}:latest</name>
+                                    <alias>${docker.testing.container}</alias>
+                                    <run>
+                                        <network>
+                                            <mode>host</mode>
+                                        </network>
+                                        <ports>
+                                            <port>4040:4040</port>
+                                        </ports>
+                                        <wait>
+                                            <time>100000</time>
+                                            <healthy>true</healthy>
+                                        </wait>
+                                        <env>
+                                            <DatabaseConfigType>${env.DatabaseConfigType}</DatabaseConfigType>
+                                            <DatabaseConfigEmbedDriver>${env.DatabaseConfigEmbedDriver}</DatabaseConfigEmbedDriver>
+                                            <DatabaseConfigEmbedUrl>${env.DatabaseConfigEmbedUrl}</DatabaseConfigEmbedUrl>
+                                            <DatabaseConfigEmbedUsername>${env.DatabaseConfigEmbedUsername}</DatabaseConfigEmbedUsername>
+                                            <DatabaseConfigEmbedPassword>${env.DatabaseConfigEmbedPassword}</DatabaseConfigEmbedPassword>
+                                            <DatabaseUsertableQuote>${env.DatabaseUsertableQuote}</DatabaseUsertableQuote>
+                                        </env>
```

Comment:
Need to be careful here.

airsonic-advanced can [read properties from the env](https://github.com/airsonic-advanced/airsonic-advanced/pull/38), which airsonic can't. I doubt these are actually setting anything inside the container. You're not using an external db anyway, so they aren't needed anyway (which is probably why your tests are fine).

Reply:
Good point, I'll remove them for now.